contrib/python-wrapper.py

```diff
@@ -19,6 +19,7 @@
 # The version tagged as 0.8.5 (download link below) builds a library with this name.
 # https://github.com/raboof/nethogs/archive/v0.8.5.tar.gz
 LIBRARY_NAME = 'libnethogs.so.0.8.5'
+LIBRARY_NAME = '/home/jantman/GIT/nethogs/src/libnethogs.so.0.8.5-32-ge545524'
```

Comment:
You probably didn't mean to check this in :)

Reply:
Eek! No, I certainly didn't!

If it's cool with you, I'll fix that and then squash down to one commit, so that's not in the history.

Reply:
Fixed that one.